data/campaigns/The_Rise_Of_Wesnoth/scenarios/17c_Troll_Hole.cfg

```diff
@@ -136,14 +131,31 @@
         {LOYAL_UNIT 4 (Tentacle of the Deep) 11 15}
         {LOYAL_UNIT 5 (Tentacle of the Deep) 19 17}
 #endif
-#ifdef HARD
 
+#ifdef HARD
         {LOYAL_UNIT 4 (Tentacle of the Deep) 11 15}
         {LOYAL_UNIT 5 (Tentacle of the Deep) 19 17}
         {LOYAL_UNIT 2 (Tentacle of the Deep) 11 17}
         {LOYAL_UNIT 3 (Tentacle of the Deep) 19 15}
 #endif
 
+        # Both tags need a "+" here in order to work:
+#define SPIDER_TRAIT
+        [+unit]
+            [+modifications]
+#ifdef EASY
+                {TRAIT_AGED}
+#endif
+#ifdef NORMAL
+                {TRAIT_SLOW}
+#endif
+#ifdef HARD
+                {TRAIT_RESILIENT}
+#endif
+            [/modifications]
+        [/unit]
+#enddef
```

Comment:
Please run the wmlindent tool on this file. The build failure is that it wants the lines inside this `#define` to be indented one tab less than they are, which 

There should be a corresponding `#undef` too.

Reply:
ok 20f8e09 ought to address this